packages/rainbowkit/src/components/ConnectButton/ConnectButton.tsx

```diff
@@ -269,7 +269,7 @@ export function ConnectButton({
                 transition="default"
                 type="button"
               >
-                {mounted && label === 'Connect Wallet'
```

Comment:
Was this necessary? Tests started failing with the refactored `useIsMounted` hook

Reply:
Aha yes i remember why this change was made. If we remove this then there will be a NextJS hydration error since on server side HTML with i18n is different than what's in the client side. 

On SSR it might be "Connect Wallet" and then if user uses spanish it will be "Conectar la billetera" which will throw hydration error.

Reply:
@DanielSinclair It's fine for now since i don't see any hydration error on my side at least. Great job on the `useIsMounted` refactoring 🙏